### ISSUE TITLE
Containers stopped randomly after being imported to casaos

### ISSUE THREAD

**Issue by AuthorShin** [IceWhaleTech/CasaOS#271]
Hello
In the new update `v3.1.1` after importing a container (stack) that has been created outside the casaos, it will stop randomly without any reason.
My only app that I have installed outside casaos and imported, is [Ghostfolio](https://github.com/ghostfolio/ghostfolio) which stops randomly and I cannot start it again unless I restart the whole system.
P.S. before importing it to casaos it was working fine.

**Comment by AuthorShin** [comment]
@JohnGuan No, cause in casaOS when they are off you cannot see the logs.

**Comment by AuthorShin** [comment]
@JohnGuan There are no error or problem in the logs of containers that are randomly stopped.

**Comment by JohnGuan** [comment]
@jerrykuku Do you have any ideas for getting information?

**Comment by AuthorShin** [comment]
@JohnGuan
I think the problem is it was a stack having 3 containers like:
ghostfolio-development_ghostfolio_1
ghostfolio-development_postgres_1
ghostfolio-development_redis_1
and when I imported it with casaOS I change the name from ghostfolio-development_ghostfolio_1 to Ghostfolio and after that i stopped working.

**Comment by JohnGuan** [comment]
So it's a problem after you changed the container name manually, right?

**Comment by AuthorShin** [comment]
@JohnGuan 
Yeah, But I was expecting the App name in casaOS was different from the actual container name, so that we can change and modify it.

**Comment by JohnGuan** [comment]
This renaming issue we have confirmed will be adjusted
